src/reference/asciidoc/rmi.adoc

```diff
@@ -75,3 +75,44 @@ The following code snippet shows the different configuration for an outbound rmi
     remote-channel="testChannel"
     host="localhost"/>
 ----
+
+=== Configuring with Java Configuration
+
+[source, java]
+----
+@Bean
+public RmiInboundGateway inbound() {
+    RmiInboundGateway gateway = new RmiInboundGateway();
+    gateway.setRequestChannel(requestChannel());
+    gateway.setRegistryHost("host");
+    gateway.setRegistryPort(port);
+    return gateway;
+}
+
+@Bean
+@ServiceActivator(inputChannel="inChannel")
+public RmiOutboundGateway outbound() {
+    RmiOutboundGateway gateway = new RmiOutboundGateway("rmi://host:port/"
+        + RmiInboundGateway.SERVICE_NAME_PREFIX + "remoteChannelName");
+    return gateway;
+}
+----
+
+Starting with _version 4.3_, the outbound gateway has a second constructor that takes a `RmiProxyFactoryBeanConfigurer` bean as a second argument.
```

Comment:
Nit-pick, but that isn't a fact that `RmiProxyFactoryBeanConfigurer` always should be a bean.
How about this:
> has a second constructor that takes a `RmiProxyFactoryBeanConfigurer` instance alongside with the service `url` argument.

?